modules/access-cluster.adoc

```diff
@@ -1,28 +1,25 @@
 // Module included in the following assemblies:
 //
-// * assemblies/accessing-cluster.adoc
-// * assemblies/quickstart-osd.adoc
+// * osd_quickstart/osd-quickstart.adoc
+// * identity_providers/config-identity-providers.adoc
 
 :_content-type: PROCEDURE
 [id="access-cluster_{context}"]
 = Accessing your cluster
 
-
-After you have configured your identity providers, users can access the cluster from the {OCM}.
+After you have added a user to your configured identity provider, you can log into your {product-title} cluster through the web console.
 
 .Prerequisites
 
-* You have created a cluster.
-* Identity providers have been configured for your cluster.
+* You logged in to {OCM}.
+* You created an {product-title} cluster.
+* You configured an identity provider for your cluster.
+* You added your user account to the configured identity provider.
 
 .Procedure
 
-. From {console-redhat-com}, click on the cluster you want to access.
-
-. Click *Open Console*.
-
-. Click on your identity provider and provide your credentials to log into the cluster.
+. Navigate to {console-redhat-com} and select your cluster.
 
-.Verification
+. Click *Open Console* to open the web console for your cluster.
```

Comment:
It's "Open console" rather than `Open Console`

Reply:
@xueli181114 thank you for your detailed review and for highlighting this. Resolved.